models/Tag.js

```diff
@@ -38,26 +27,9 @@ module.exports = function(sequelize, DataTypes) {
     trapType: {
       type: DataTypes.STRING,
     },
-    // Time in the recording that the pest started interacting with the trap
-    trapInteractionTime: {
-      type: DataTypes.FLOAT,
-    },
-    // How long the trap is interacted for before triggering
-    trapInteractionDuration: {
-      type: DataTypes.FLOAT,
-    },
-    trappedTime: { // If the animal got trapped, time in the recording.
-      type: DataTypes.FLOAT,
-    },
-    killedTime: { // If the animal got killed, time in the recording.
-      type: DataTypes.FLOAT,
-    },
-    poisionedTime: { // If the animal got poisioned, time in the recording.
-      type: DataTypes.FLOAT,
+    sex: { // What sex is the animal, null if don't know.
+      type: DataTypes.ENUM('F', 'M'),
```

Comment:
There's no way we're going to be able to answer this from the videos... should this just go?